Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/GltfEditorImporter.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema;
+using System.IO;
+using UnityEditor;
+using UnityEditor.Experimental.AssetImporters;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization.Editor
+{
+    public static class GltfEditorImporter
+    {
+        public static async void OnImportGltfAsset(AssetImportContext context)
+        {
+            var gltfAsset = (GltfAsset)ScriptableObject.CreateInstance(typeof(GltfAsset));
+            var importedObject = await GltfUtility.ImportGltfObjectFromPathAsync(context.assetPath);
+
+            gltfAsset.GltfObject = importedObject;
```

Comment:
ImportGltfObjectFromPathAsync can return null, in which case we should probably return prior to any of this logic

Reply:
picking up this fix